tests/integration/components/adde-tooltip-icon-test.js

```diff
@@ -112,3 +112,32 @@ test('tooltip box direction can be modified', async function(assert) {
     'tooltip box reflects correct direction'
   );
 });
+
+test('tooltip can set iconWeight', async function(assert) {
+  assert.expect(2);
+
+  this.render(hbs`
+    {{#adde-tooltip-icon data-test-tooltip-icon=true iconWeight="far"}}
```

Comment:
```suggestion
    {{#adde-tooltip-icon data-test-tooltip-icon=true iconWeightClass="far"}}
```

? I think the argument is `iconWeightClass`?